encounters/README.md

```diff
@@ -114,6 +124,8 @@ An GP Connect FHIR `Encounter` is mapped from an GP2GP HL7v3 `EHR Composition`.
 1. If a SNOMED CT code cannot be found `type[0].coding` will not be populated.
 2. Where the participant is the Practitioner that recorded the consultation on the system, identified by `ehrComposition / author`.  
 3. Populated only where a `ehrComposition / participant2` is populated.
+4. If a `confidentialityCode` Coded Value (CV) containing code `NOPAT` **CAN** be found within the `ehrComposition` then the `meta.security` **SHALL** be added.
+5. If a `confidentialityCode` Coded Value (CV) containing code `NOPAT` **CANNOT** be found within the `ehrComposition` then the `meta.security` **SHALL NOT** be added.
```

Comment:
These numbered items relate to the `<sup>n</sup>` above, given that I can't see a `<sup>4</sup>` or `<sup>5</sup>` I don't think you're using them right 😄 .